store/src/java/com/zimbra/qa/unittest/TestDeployZimlet.java

```diff
@@ -48,7 +48,7 @@
 import com.zimbra.soap.admin.type.AttachmentIdAttrib;
 
 /**
- * copy test data from data/unittest to /opt/zimbra/data/unittest before running this test 
+ * copy test data from data/unittest to /opt/zimbra/unittest before running this test 
```

Comment:
was thinking the same thing. I hesitate creating an ANT target for this yet, but we will need it eventually.